src/framework/components/particlesystem/particlesystem_system.js

```diff
@@ -195,12 +195,49 @@ pc.extend(pc, function() {
                 type: 'boolean',
                 defaultValue: false
             }, {
-                name: "spawnBounds",
-                displayName: "Spawn Bounds",
-                description: "Defines an AABB in which particles are allowed to spawn",
+                name: "emitterShape",
+                displayName: "Emitter Shape",
+                description: "Defines a shape in which particles are allowed to spawn",
+                type: "enumeration",
+                options: {
+                    enumerations: [{
+                        name: 'Box',
+                        value: pc.EMITTERSHAPE_BOX
+                    }, {
+                        name: 'Sphere',
+                        value: pc.EMITTERSHAPE_SPHERE
+                    }]
+                },
+                defaultValue: pc.EMITTERSHAPE_BOX,
+            }, {
+                name: "emitterExtents",
+                displayName: "Emitter Extents",
+                description: "Defines a local box in which particles are allowed to spawn",
                 type: "vector",
+                filter: {
+                    emitterShape: pc.EMITTERSHAPE_BOX
+                },
                 defaultValue: [0, 0, 0]
             }, {
+                name: "emitterRadius",
+                displayName: "Emitter Radius",
+                description: "Defines radius in which particles are allowed to spawn",
+                type: "number",
+                filter: {
+                    emitterShape: pc.EMITTERSHAPE_SPHERE
+                },
```

Comment:
Should the default value for emitterRadius be something more meaningful like 1 ?